remsfal-service/src/main/java/de/remsfal/service/boundary/project/PropertyResource.java

```diff
@@ -42,4 +43,22 @@ public PropertyJson getProperty(final String projectId, final String propertyId)
         return PropertyJson.valueOf(model);
     }
 
+    @Override
+    public PropertyListJson getProperties(String projectId, Integer offset, Integer limit) {
+        List<PropertyModel> properties = controller.getProperties(projectId, offset, limit);
+        return PropertyListJson.valueOf(properties, offset, properties.size());
+    }
+
+    @Override
+    public void deleteProperty(String projectId, String propertyId) {
+        checkPrivileges(projectId);
```

Comment:
Isn't this privilege check also required for existing methods?

Reply:
Done